xeeg_hed_score/sub-eegSeizureTUH/ses-eeg01/eeg/sub-eegSeizureTUH_ses-eeg01_channels.tsv

```diff
@@ -0,0 +1,23 @@
+name	type	units	reference

```

Comment:
Units should have an interpretable value

Reply:
Done